screenpro/phenoscore/__init__.py

```diff
@@ -56,6 +56,9 @@ def runPhenoScore(adata, cond_ref, cond_test, score_level, var_names='target', t
     """
     adat = adata.copy()
 
+    if 'condition' not in adat.obs.columns:
```

Comment:
This works for now, thanks for adding this as a more clear error message. We may change it in future so `runPhenoScore` can take "condition" as an input.